source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+the large job. While this does make more efficient use of the system, it
+encourages the submission of smaller jobs.
 
-  > module help gcp
 
-  ----------- Module Specific Help for 'gcp/2.2' --------------------
+Job priority
+------------
 
-  Sets up the shell environment for gcp
+Slurm on Gaea uses the `Slurm priority/multifactor plugin
+<https://slurm.schedmd.com/priority_multifactor.html>`_ to calculate a job's
+priority.  The factors used are:
 
+Age
+    the length of time ajob has been waiting in the queue, elgible to be
+    scheduled
 
-**module avail:** List all available modulefiles in the current
-MODULEPATH.
+Fair-share
+    the difference between the portion of the computing resource that has been
+    promised (allocation) and the amount of resources that has been consumed.
+    Gaea uses the `classic fairshare algorithm
+    <https://slurm.schedmd.com/classic_fair_share.html>`_
 
-.. code-block:: shell
 
-  ------------------------------------------ /opt/cray/ss/modulefiles ---------------------------------------
-  portals/2.2.0-1.0301.22039.18.1.ss(default) rca/1.0.0-2.0301.21810.11.20.ss(default)
-  ------------------------------------------ /opt/cray/xt-asyncpe/default/modulefiles -----------------------
-  xtpe-accel-nvidia20  xtpe-barcelona       xtpe-istanbul        xtpe-mc12            xtpe-mc8             xtpe-network-gemini
-  xtpe-network-seastar xtpe-shanghai        xtpe-target-native
-  ------------------------------------------ /opt/cray/modulefiles ------------------------------------------
-  atp/1.0.2(default)                   perftools/5.1.0(default)             portals/2.2.0-1.0300.20621.14.2.ss   trilinos/10.2.0(default)
-  atp/1.1.1                            perftools/5.1.2                      rca/1.0.0-2.0300.20198.8.26.ss       trilinos/10.6.2.0
-  ga/4.3.3(default)                    pmi/1.0-1.0000.7628.10.2.ss          rca/3.0.20                           xt-mpich2/5.0.1(default)
-  gdb/7.2(default)                     pmi/1.0-1.0000.7901.22.1.ss(default) stat/1.0.0(default)                  xt-mpich2/5.2.0
-  iobuf/2.0.1(default)                 pmi/1.0-1.0000.8256.50.1.ss          stat/1.1.3                           xt-mpt/5.0.1(default)
-  xt-mpt/5.2.0                         xt-shmem/5.0.1(default               xt-shmem/5.2.0
+:abbr:`QOS (Quality of Service)`
+    a factor associated with each Quality Of Service (QOS)
 
 .. note::
 
-  Your shell might print out something more, or something different.
-
-**module add module_file:** Load module file(s) into the shell
-environment
-
-**module load module_file:** Load module file(s) into the shell
-environment
-
-.. code-block:: shell
-
-  > module load gcp/1.1
-
-
-**module list:** List of Loaded modules.
+    Only the compute clusters' QOSes will affect a job's priority value.  ES
+    cluster QOSes (the login nodes and the DTNs) QOSes specific to the login
+    nodes and DTNs do not affect the job's prioities.
+
```

Comment:
I don't know what you meant to say, but I'm confused by 
"ES  cluster QOSes (the login nodes and the DTNs) QOSes"

Reply:
Adjusted in a new push.